building/tracks/new/setup-continuous-integration.md

```diff
@@ -8,9 +8,9 @@ GitHub actions uses the concept of _workflows_, which are scripts that are run a
 Each workflow corresponds to a file in `.github/workflows`.
 Each new track repository comes pre-loaded with three workflows:
 
-- `ci.yml`: this workflow should run the tests for each exercise the track has implemented
+- `test.yml`: this workflow should run the tests for each exercise the track has implemented
```

Comment:
This doesn't match any existing files, templates, or existing docs. It's called `ci.yml` everywhere.

Reply:
Some workflows are about repo automation and management and the likes, I'd not consider those CI. But I see your point.

I'd suggest you also update the names in other places where `ci.yml` occurs in the docs, like here: https://github.com/exercism/docs/blob/main/building/tracks/ci/workflow-templates.md (probably better in a separate PR, though)

Reply:
Yeah I'd like that. I'm also simplifying the workflows a bit.